### DISCUSSION TITLE
Well integrated mail apps

### DISCUSSION THREAD

**Discussion by halb9** [elementary/streams#38] [Ideas]
What is the current state of a well integrated mail app? 
Geary was 'rudimentary', at best. 
All others like Evolution, Thunderbird etc. do not fit into the eOS style guidlines. 
I would love to hear a quick introduction what "good" suggestions there are for a mail client. 

Many thanks!

**Comment by hanaral** [comment]
Elementary mail does not really = geary. There is a 2.0 in progress, which brings a huge amount of system integration, like moving all the account configuration to the onlineaccounts panel and UX changes like actual Dark Style support.
